docs/index.asciidoc

```diff
@@ -0,0 +1,231 @@
+:plugin: google_pubsub
+:type: output
+:default_codec: plain
+
```

Comment:
The`lib/logstash/outputs/google_pubsub.rb` file says the default codec is `json`.  If that's true, then set it here. 
`:default_codec: json`

Reply:
Whoops, I knew I had to change it somewhere and added it at the bottom of the file instead :man_facepalming: